### DISCUSSION TITLE
Network strength vs speed, both real-time?

### DISCUSSION THREAD

**Discussion by anvoice** [VREMSoftwareDevelopment/WiFiAnalyzer#401] [Q&A] [answered]
Hello, I was wondering how accurate the network speed shown in the app channel graph is? Does the app actually measure the network transmission speed between the device and router? I noticed that the "speed" for my network varies in strange ways in relation to the "signal strength": I am walking around with the phone, and getting plausibly varying signal strength but the speed can stay the same low (e.g. 50Mbps between -36 and -78dBm) or high value until it seemingly randomly updates. I am trying to map bad spots in the house and not knowing whether the shown signal speed reflects the actual value real-time is making it impossible. Should I simply go by signal strength instead? I did turn off wifi scan throttling on Android 13. Thanks for reading!

**Comment by VREMSoftwareDevelopment** [comment, marked as answer]
Network speed and signal strength are provided by Android API.
